skyvern/forge/sdk/executor/background_task_executor.py

```
@@ -0,0 +1,184 @@
+            )
+            background_tasks.add_task(
+                task_v2_service.run_task_v2,
+                organization=organization,
+                task_v2_id=task_v2_id,
+                max_steps_override=max_steps_override,
+                browser_session_id=browser_session_id,
+            )
+
+    async def execute_script(
+        self,
+        request: Request | None,
```

Ruff (0.14.8)

Unused method argument: `request`

(ARG002)